go.mod

```diff
@@ -3,11 +3,8 @@ module github.com/googleapis/gax-go
 go 1.11
 
 require (
-	github.com/golang/protobuf v1.3.1
+	github.com/golang/protobuf v1.3.1 // indirect
```

Comment:
Is this intentional?

Reply:
We shouldn't need to make any changes to the root `go.mod`. The `v2` module is the one we are concerned with. So please revert this change.

Reply:
`git checkout master -- <the.file>`